.github/helm/affine/charts/graphql/templates/deployment.yaml

```diff
@@ -28,10 +28,10 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
-          - name: AUTH_PRIVATE_KEY
+          - name: AFFINE_PRIVATE_KEY
             valueFrom:
               secretKeyRef:
-                name: "{{ .Values.app.jwt.secretName }}"
+                name: private-key
```

Comment:
keep the `"{{ .Values.app.jwt.secretName }}"` logic to let users override the secret name, in case they already have a secret named `private-key`

Reply:
is it possible to be shared with sync service if it's under `.Values.app`?

Reply:
You can put it into `global`: https://github.com/toeverything/AFFiNE/blob/canary/.github/helm/affine/values.yaml#L1 and share it with subcharts, like `.Values.global.jwt.secret-name`